pkg/common/morpc/client.go

```diff
@@ -378,19 +382,26 @@ func (c *client) createTask(ctx context.Context) {
 	}
 }
 
-func (c *client) createBackend(backend string) (Backend, error) {
+func (c *client) createBackend(backend string, lock bool) (Backend, error) {
```

Comment:
Is this function doing same thing as getBackend?     Should we merge these as same function?

Reply:
getBackend tries to get an available backend to return to the client,  and if none is available it calls createBackend to create one.